fastify.js

```diff
@@ -353,7 +353,8 @@ function fastify (options) {
     validateSchemaErrorFormatter(options.schemaErrorFormatter)
     fastify[kSchemaErrorFormatter] = options.schemaErrorFormatter.bind(fastify)
   }
-
+  
+  const avvioPluginTimeout = Number(options.pluginTimeout);
```

Comment:
```suggestion
  const avvioPluginTimeout = Number(options.pluginTimeout)
```
No need for a semicolon here

Reply:
The linter sniffed that out. Thanks. I think I changed it already.